Telegram/server.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const projectId = 'my-project';
+// const location = 'global';
+// const agentId = 'my-agent';
+// const languageCode = 'en'
+// const TELEGRAM_TOKEN='1234567898:ABCdfghTtaD8dfghdfgh45sdf65467M';
+// const SERVER_URL='https://example.com';
+
+const express = require('express');
+const axios = require('axios');
+const bodyParser = require('body-parser');
+
+const API_URL = `https://api.telegram.org/bot${TELEGRAM_TOKEN}`;
+const URI = `/webhook/${TELEGRAM_TOKEN}`;
+const WEBHOOK = SERVER_URL + URI;
+
+const app = express();
+app.use(bodyParser.json());
+
+
+// Imports the Google Cloud Some API library
+const {SessionsClient} = require('@google-cloud/dialogflow-cx');
+/**
+ * Example for regional endpoint:
+ *   const location = 'us-central1'
+ *   const client = new SessionsClient({apiEndpoint: 'us-central1-dialogflow.googleapis.com'})
+ */
+const client = new SessionsClient();
+
+/**
+ * Calls the Dialogflow CX API with the specific user query input and returns a string
+ * from the API with a response for the users query depending on the matched intent.
+ */
+async function detectIntentText(query) {
+  let agentResponse = '';
+
+  const sessionId = Math.random().toString(36).substring(7);
+  const sessionPath = client.projectLocationAgentSessionPath(
+      projectId,
+      location,
+      agentId,
+      sessionId
+  );
+  console.info(sessionPath);
+
+  const request = {
+    session: sessionPath,
+    queryInput: {
+      text: {
+        text: query,
+      },
+      languageCode,
+    },
+  };
+  const [response] = await client.detectIntent(request);
+  console.log(`User Query: ${query}`);
+  for (const message of response.queryResult.responseMessages) {
+    if (message.text) {
+      console.log(`${message.text.text}`);
+      agentResponse += `${message.text.text}\n`;
+    }
+    
+  }
+  if (response.queryResult.match.intent) {
```

Comment:
nit: I guess this is just for debugging? If so, you may consider add a comment to mention that.